grype/db/v6/installation/curator.go

```diff
@@ -443,6 +467,12 @@ func (c curator) Import(path string) error {
 	return nil
 }
 
+var urlPrefixPattern = regexp.MustCompile("^[a-zA-Z]+://")
```

Comment:
It looks like this will match `file://`; I think this will end up working fine, since it's supported by go-getter, is that right?

Reply:
yup yup -- it supports handling `file://` 👍